webrtc-audio-processing-sys/build.rs

```diff
@@ -66,6 +67,13 @@ mod webrtc {
     fn copy_source_to_out_dir() -> Result<PathBuf, Error> {
         use fs_extra::dir::CopyOptions;
 
+        if Path::new(BUNDLED_SOURCE_PATH).read_dir()?.next().is_none() {
+            eprintln!("The webrtc-audio-processing source directory is empty.");
+            eprintln!("See the crate README for installation instructions.");
+            eprintln!("Remember to clone the repo recursively if building from source.");
+            panic!("Aborting compilation because bundled source directory is empty.")
```

Comment:
It looks like it would be possible to return an error instead? That would bubble up to `main()`, and cargo hopefully interprets that.

Reply:
I've changed the panic to a bail. Let me know if you had something else in mind.

Reply:
bail is good here, thanks.